### PR TITLE
Report total lines of code in CI as a lint pass

### PR DESCRIPTION
**Pull request by leo-lb**
I would like it if that count could also be posted as an automatic comment for better visibility but it doesnt look like that's possible.

### CODE REVIEW

**Comment by leo-lb** [comment]
So right now we've got:

```
         Language  Files  Lines  Blanks  Comments  Code
         --------  -----  -----  ------  --------  ----
         Rust      21     1603   231     375       997
         --------  -----  -----  ------  --------  ----
Totals:            21     1603   231     375       997
```
https://github.com/stjepang/smol/runs/625722633#step:4:4

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -12,3 +12,15 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install cargo-count
+        run: cargo install cargo-count
+
+      - name: Run cargo count
+        # '-a --exclude=$(cat .gitignore)' necessary to work around:
+        # https://github.com/kbknapp/cargo-count/issues/36
+        run: cargo count -l rs -a --exclude=$(cat .gitignore) src
```
Comment (stjepang) [inline .github/workflows/lint.yaml:26]: nit: missing newline :)

Comment (leo-lb) [reply .github/workflows/lint.yaml:26]: I can add it, but Github Actions seems to be fine with it, necessary?

Comment (stjepang) [reply .github/workflows/lint.yaml:26]: Not necessary but it's good to always have the trailing newline (some editors and CLI tools don't like it when the newline is missing).

**Comment by stjepang** [comment]
If you'd like to verify whether the LOC in the readme matches the actual LOC, I've managed to extract the lines of code like this:

```
$ cargo count -l rs -- $PWD/src | head -4 | tail -1 | tr -s ' ' | cut -d' ' -f7
1568
```

Then we can grep README (it's okay to reformat it a bit) and warn if the difference is <100.
I wouldn't make it an error in CI though because I worry that might get a bit too annoying.

**Comment by leo-lb** [comment]
@stjepang I don't feel like it's necessary to do this. We know the line count written in the README, it's 1500. It'll probably be that way for a while, so everyone will know, too. If we warn inside the action log, then it means someone went through the effort of visiting that log in the first place, probably because they wanted to compare the numbers themselves already. I think this kind of check makes sense only if it is posted as a comment for better visibility.

**Comment by stjepang** [comment]
Got it, I like this! Let's merge.

**Pull request merged**